TopPlaces/TopPlaces/Model/PlacesPhotosProvider.h

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2016 Lightricks. All rights reserved.
```

Comment:
Please split this file.
<code>CancellableTask</code> can remain here but the Place/PhotoInfo protocol should be extracted.